src/utils/helpers.ts

```diff
@@ -0,0 +1,17 @@
+import { Todo } from '../types/Todo';
+import { TaskStatus } from '../types/Sort';
+
+export const getFilteredTodos = (todos: Todo[], sortType: TaskStatus) => {
+  return todos.filter(todo => {
+    switch (sortType) {
+      case TaskStatus.ACTIVE:
+        return !todo.completed;
+
+      case TaskStatus.COMPLETED:
+        return todo.completed;
+
+      default:
+        return todos;
```

Comment:
why do we need to return all todos on each iteration?

just check your sortType

```
if (sortType === TaskStatus.ALL) {
  return todos;
}

return todos.filter(el => ...
```